### PR TITLE
CNV-9055: RN - Added note about NMState IP management support

### PR DESCRIPTION
**Pull request by sjhala-ccs**
This PR addresses [CNV-9055](https://issues.redhat.com/browse/CNV-9055)

Added note about Kubernetes NMState IP configuration options

Preview build: https://deploy-preview-33479--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes.html#virt-4-8-networking-new

### CODE REVIEW

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -58,6 +58,7 @@ The SVVP Certification applies to:
 //CNV-9054 MAC address pool is now enabled on all namespaces by default.
 
 //CNV-9055 Kubernetes NMState now supports new IP configuration options.
+* You can use the Kubernetes NMSstate Operator to xref:../virt/node_network/virt-updating-node-network-config.adoc#virt-example-nmstate-IP-management_virt-updating-node-network-config[configure and manage IP addresses] on your cluster node network.
```
Comment (phoracek) [inline virt/virt-4-8-release-notes.adoc:61]: I would drop "network" from "node network". It seems redundant and slightly confusing - you don't manage the whole node network (e.g. you cannot control IP ranges of network HW such as routers). You can really only control IP addresses of nodes.

Comment (sjhala-ccs) [reply virt/virt-4-8-release-notes.adoc]: Removed the word "network"

**Review by phoracek — APPROVED** [review]
Looks good, thanks

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**